### PR TITLE
Turn a few verb-only pytest fixtures into standard functions

### PR DESCRIPTION
**Pull request by dralley**
[noissue]

### CODE REVIEW

**Comment by dralley** [comment]
@mdellweg 

>Probably not if someone started using it.  That is until pulpcore 3.55.

I don't think we provide semantic versioning guarantees for test infrastructure?  And any time we do test infra changes we often end up backporting it anyway, just to avoid incompatibility when future tests are added.

>None at all. Just use requests.get.

IMO, if we do that, and just eat the extra dependency, then we might as well just delete the function entirely.  There's no point having an alias for a function as simple as `requests.get`

**Comment by gerrod3** [comment]
I would say deprecate the fixtures, copy them to the utils and slowing switch tests over to importing the util functions. Once all the plugins have switched, then we can remove the fixtures.

**Comment by mdellweg** [comment]
> I would say deprecate the fixtures, copy them to the utils and slowing switch tests over to importing the util functions. Once all the plugins have switched, then we can remove the fixtures.

It's the lower bounds test that makes me think about proper deprecation and removal only in a breaking change release.

File: pulpcore/tests/functional/__init__.py [hunk 1]
```diff
@@ -1060,7 +1060,6 @@ def _add_to_filesystem_cleanup(path):
                 pass
 
 
-@pytest.fixture(scope="session")
```
Comment (mdellweg) [inline pulpcore/tests/functional/__init__.py:1063]: I don't think we want this to stop being a fixture. It is using other fixtures.

**Pull request closed without merging**